jmxtrans-core/src/test/java/com/googlecode/jmxtrans/JmxTransformerTest.java

```diff
@@ -46,8 +46,8 @@
 
 @AllowDNSResolution
 @AllowLocalFileAccess(paths = "*")
-@Category({IntegrationTest.class, RequiresIO.class})
```

Comment:
The correct fix should be to configure [Maven Failsafe plugin](https://maven.apache.org/surefire/maven-failsafe-plugin/). I'd like to make sure that unit tests are actually testing small units and are fast. As we are also using [PIT](http://pitest.org/), it is specially important that unit tests are fast.

Could you rollback this specific change and open an issue to add an integration test phase to the build ?